Wikipedia/UI-V5/WMFAppViewController.m

```diff
@@ -362,4 +360,14 @@ - (void)searchLanguageDidChangeWithNotification:(NSNotification*)note {
     [self configureSearchViewController];
 }
 
+#pragma mark - UINavigationControllerDelegate
+
+- (void)navigationController:(UINavigationController*)navigationController
+      willShowViewController:(UIViewController*)viewController
+                    animated:(BOOL)animated {
+    BOOL isToolbarEmpty = [viewController toolbarItems].count == 0;
+    [self.rootTabBarController wmf_setTabBarVisible:isToolbarEmpty animated:NO completion:nil];
```

Comment:
I can try to add that back, but IIRC there were issues w/ it. Unfortunately I forgot to document them :disappointed: